### DISCUSSION TITLE
How do I run external command via the Cmdline interface?

### DISCUSSION THREAD

**Discussion by Pilgrim1379** [LazyVim/LazyVim#888] [Q&A] [answered]
I'm trying use the shells `mkdir` command to create a directory. So I type:
`:` to bring up the `Cmdline` interface which pops up at the top of my window.
Then I type `!` on my macbook expecting to get the exclamation sign so I could use the `mkdir` command to create a directory.
Instead what I get is the `$` and the window changes to `Filter`.
How can i run external shell command in Lazyvim?

**Comment by folke** [comment, marked as answer]
Just type your comamnd `:!` is concealed as `$`. Similar to a command prompt :)

**Reply by Pilgrim1379** [reply]
> Just type your comamnd `:!` is concealed as `$`. Similar to a command prompt :)

Oh I see! The change from `Cmdlne` to `Filter` threw me :)
